elearning-backend/app/Http/Controllers/API/AdminWordsController.php

```diff
@@ -53,28 +54,57 @@ public function store(CreateWordsPostRequest $request)
     {
         // Convert to JSON string for storing
         $choices = json_encode($request->choices);
+        $correctAnswer = $this->getCorrectAnswer($request->choices);
+        
+        if ($correctAnswer === "") {
+            throw new \ErrorException('No correct answer has been set!');
+            return;
+        }
+
+        Word::create([
+            'category_id' => $request->category_id,
+            'choices' => $choices,
+            'correct_answer' => $correctAnswer,
+            'word' => $request->word,
+        ]);
+    }
+
+    // Get correct answer from array
+    private function getCorrectAnswer($choices)
+    {
         $correctAnswer = "";
 
-        // Get correct answer
-        foreach($request->choices as $key) {
+        foreach($choices as $key) {
             if ($key['choice'] === null ) {
                 throw new \ErrorException('Please set all choices!');
                 return;
             }
 
             $correctAnswer = $key['is_correct'] ? $key['choice'] : $correctAnswer;
         }
+
+        return $correctAnswer;
+    }
+
+    // Update word
+    public function update($id, CreateWordsPostRequest $request)
+    {
+        // Convert to JSON string for storing
```

Comment:
pls remove comments. just give meaningful variable / function name. If we will put comments, meaning our given name is not understandable.
We will just add comments if logic is complex